src/Oro/Bundle/UserBundle/Entity/UserManager.php

```diff
@@ -314,4 +326,71 @@ public function getApi(User $user, Organization $organization)
     {
         return $this->getStorageManager()->getRepository('OroUserBundle:UserApi')->getApi($user, $organization);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function loadUserByOAuthUserResponse(UserResponseInterface $response)
+    {
+        if (!$this->cm->get('oro_user.enable_google_sso')) {
+            throw new \Exception('SSO is not enabled');
+        }
+
+        $username = $response->getUsername();
+        if ($username === null) {
+            throw new AccountNotLinkedException(sprintf("User '%s' not found.", $username));
+        }
+
+        $user = $this->findUserBy(array($this->getOAuthProperty($response) => $username));
```

Comment:
I saw that you're using short array syntax just sometimes (probably newer code?) so I wrote it according to the class.

So you're using short syntax for all new code in all places?